maps.c

```diff
@@ -119,7 +119,13 @@ bool readmaps(pid_t target, list_t * regions)
                  * There should never be more than these four regions.
                  * The data regions use their variables relative to the load
                  * address. So determining it makes sense as we can get the
-                 * variable address used within the binariy with it.
+                 * variable address used within the binary with it.
+                 *
+                 * 04/2015: Usually there is nothing in between these four
```

Comment:
Instead of appending the log, I suggest you to update the existing ones to reflect the current situation.